lisa/tools/dhclient.py

```diff
@@ -44,15 +48,14 @@ def get_timeout(self) -> int:
                 value = int(group["number"])
                 is_default_value = False
         elif isinstance(self.node.os, Fedora):
-            # the default value in fedora is 60
-            value = 60
-            # use cat to output all together
-            result = self.node.execute(
-                "nmcli connection show 'System eth0' | cat", shell=True
-            )
-            group = find_group_in_lines(result.stdout, self._fedora_pattern)
-            if group and "default" not in group:
-                value = int(group["number"])
+            # the default value in fedora is 45
+            value = 45
+            result = self.node.tools[Journalctl].logs_for_unit("NetworkManager")
+            groups = find_groups_in_lines(result, self._fedora_pattern)
+            if len(groups) == 0:
+                raise LisaException("not find matched dhcp timeout line")
+            if value != int(groups[-1]["number"]):
```

Comment:
Why there are multiple items after regexp? if so, should we check each line, including both ipv4 and ipv6?

Reply:
find another better way to check config using `NetworkManager --print-config`, will update code and test